INSTALL

```diff
@@ -501,9 +506,9 @@ For the QGIS build you need to install following packages from cygwin:
 
 - bison
 - flex
-- git
+- git (even if you already have Git for Windows installed)
 
-and from OSGeo4W (select Advanced Installation):
+and from OSGeo4W (select Advanced Install):
 
 - qgis-dev-deps
```

Comment:
I think this error at startup is due to not having `owslib` in my OSGeo4W install. Apparently, this package isn't included as a dependency of [the `qgis-dev-deps` package](https://github.com/qgis/QGIS/blob/5bf120a885ff3344ae1724add38f190de152504b/INSTALL#L508)?

```
Couldn't load plugin 'MetaSearch' due to an error when calling its classFactory() method 

ModuleNotFoundError: No module named 'owslib' 
Traceback (most recent call last):
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python\qgis\utils.py", line 334, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python/plugins\MetaSearch\__init__.py", line 29, in classFactory
    from MetaSearch.plugin import MetaSearchPlugin
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python\qgis\utils.py", line 743, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python/plugins\MetaSearch\plugin.py", line 34, in 
    from MetaSearch.dialogs.maindialog import MetaSearchDialog
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python\qgis\utils.py", line 743, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python/plugins\MetaSearch\dialogs\maindialog.py", line 50, in 
    from owslib.csw import CatalogueServiceWeb # spellok
  File "C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python\qgis\utils.py", line 743, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'owslib'


Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] 
QGIS version: 3.13.0-Master Master, 838bebea83 

Python Path:
C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python
C:/Users/codyt/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/codyt/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/OSGeo4W64/QGIS/ms-windows/osgeo4w/build-qgis-test-x86_64/output/python/plugins
C:\OSGeo4W64\apps\Python37\python37.zip
C:\OSGeo4W64\apps\Python37\DLLs
C:\OSGeo4W64\apps\Python37\lib
C:\OSGeo4W64\QGIS\ms-windows\osgeo4w\build-qgis-test-x86_64\output\bin\RelWithDebInfo
C:\OSGeo4W64\apps\Python37
C:\OSGeo4W64\apps\Python37\lib\site-packages
C:\OSGeo4W64\apps\Python37\lib\site-packages\win32
C:\OSGeo4W64\apps\Python37\lib\site-packages\win32\lib
C:\OSGeo4W64\apps\Python37\lib\site-packages\Pythonwin
C:/Users/codyt/AppData/Roaming/QGIS/QGIS3\profiles\default/python
```

It doesn't prevent me from using QGIS to do the training exercises, so maybe leaving it out is intentional.

Should we be telling users to install the `owslib` and `python3-owslib` libraries as part of their custom OSGeo4W install? We [warn them against doing it right now](https://github.com/qgis/QGIS/blob/5bf120a885ff3344ae1724add38f190de152504b/INSTALL#L512).

Reply:
> It doesn't prevent me from using QGIS to do the training exercises, so maybe leaving it out is intentional.

It's not a build dependency.

Reply:
> > It doesn't prevent me from using QGIS to do the training exercises, so maybe leaving it out is intentional.
> 
> It's not a build dependency.

👍 then I think this is ready to merge.